### ISSUE TITLE
Question：Can the messages of dreamview websocket be deserialized by "Object.Parser.ParseFrom(stream)"  directly？

### ISSUE THREAD

**Issue by ilovevision** [ApolloAuto/apollo#13073]
### System information

- **OS Platform and Distribution (e.g., Linux Ubuntu 18.04)**:ubuntu18.04
- **Apollo installed from (source or binary)**:source
- **Apollo version (3.5, 5.0, 5.5, 6.0)**:6.0

### Steps to reproduce the issue:
- **Please use bullet points and include as much details as possible**
I want to deserialize the messages grasped by websocket，Can it be deserialized by protobuf ？as follows：
![2020-11-27 16-31-55 的屏幕截图](https://user-images.githubusercontent.com/17542461/100428236-a126c180-30ce-11eb-9ed4-58a5f1b9f270.png)
:

**Comment by ntutangyun** [comment]
They are probably protobuf binary data, you have to deserialize with protobuf javascript library.

**Comment by ilovevision** [comment]
Isn't protobuf cross-platform?

**Comment by ntutangyun** [comment]
> Isn't protobuf cross-platform?

Yes sorry thought you were trying to use javascript to decode it. Essentially you could use any language that Protobuf supports. You may get those libraries from the .proto files in Apollo project.
